article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -304,13 +255,31 @@ object DotcomponentsDataModel {
         main.getOrElse(Configuration.images.fallbackLogo)
       }
 
-      NewsArticle(
-        `@id` = article.metadata.webUrl,
-        images = Seq(ImgSrc(mainImageURL, Item1200)),
-        author = article.tags.contributors.mkString(", "),
-        datePublished = article.trail.webPublicationDate.toString(),
-        dateModified = article.fields.lastModified.toString(),
-        headline = article.trail.headline,
+      val authors = article.tags.contributors.map(contributor => {
+        Person(
+          name = contributor.name,
+          sameAs = contributor.metadata.webUrl,
+        )
+      })
+
+      List(
+        NewsArticle(
+          `@id` = Configuration.amp.baseUrl + article.metadata.id,
+          images = Seq(
+            ImgSrc(mainImageURL, OneByOne),
+            ImgSrc(mainImageURL, FourByThree),
+            ImgSrc(mainImageURL, Item1200),
+          ),
+          author = authors,
+          datePublished = article.trail.webPublicationDate.toString(),
+          dateModified = article.fields.lastModified.toString(),
+          headline = article.trail.headline,
+          mainEntityOfPage = article.metadata.webUrl,
+        ),
+        WebPage(
+          `@id` = article.metadata.webUrl,
```

Comment:
Yeah, this is what I expected in this field but don't know how `newsArticle` differs